Automattic/jetpack
# CRM: Make form fonts smaller

## Proposed changes:

This PR changes our input fonts (and some others) to the default size/family used by Jetpack.

### Other information:


## Jetpack product discussion
p1HpG7-lcQ-p2


## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:






To test this do a tour around the CRM and check for inconsistencies in font sizes/family.
Most notable changes:

Edit Contact:
`wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=contact&zbsid=1`

Edit Company:
`wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=company&zbsid=1`

Edit Quotes:
`wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=quote&zbsid=1`

Edit Invoice:
`wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=invoice&zbsid=1`

Edit Transaction:
`wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=transaction&zbsid=1`

Edit Forms:
`wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=form&zbsid=1`
